setup/tinyHIPPOstart

```diff
@@ -14,21 +14,21 @@ pip3 install scapy
 pip3 install dataclasses
 pip3 install requests
 
-# set config path environment variable
-export IOT_IDS_CONFIGPATH='/etc/tinyHIPPO/config.json'
+# set install path environment variable
+export TINYHIPPO_INSTALL_PATH='/etc/tinyHIPPO/'
```

Comment:
When I attempted to install and run, the output of 'ps' showed this:
```
10549 root     21120 R    python3 /etc/tinyHIPPO//tiny_hippo_run.py
```
I don't think there needs to be two forward slashes, so change it to only one.

Reply:
Pushed these changes